### PR TITLE
Finished CRUD for Article

### PR DESCRIPTION
**Pull request by hardikupadhyay16**
- Created CRUD for article
- changed column type for long description in article model

### CODE REVIEW

File: app/views/articles/_article.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1,2 @@
+json.extract! article, :id, :created_at, :updated_at
+json.url article_url(article, format: :json)
```
Comment (KanjariyaHardik) [inline app/views/articles/_article.json.jbuilder:2]: Need Space Here

File: app/views/articles/index.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1 @@
+json.array! @articles, partial: 'articles/article', as: :article
```
Comment (KanjariyaHardik) [inline app/views/articles/index.json.jbuilder:1]: Space here at the end of file

File: app/views/articles/show.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1 @@
+json.partial! "articles/article", article: @article
```
Comment (KanjariyaHardik) [inline app/views/articles/show.json.jbuilder:1]: Space here at the end of file

**Review by KanjariyaHardik — APPROVED** [review]
Please make changes to code structure

File: app/views/articles/_article.json.jbuilder (hunk 1 above)
Comment (charusat09) [reply app/views/articles/_article.json.jbuilder:2]: @hardikupadhyay16  @uzaif313  I think we don't need this at all...

File: app/controllers/articles_controller.rb [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  def edit
+  end
+
+  # POST /articles
+  # POST /articles.json
+  def create
+    @article = current_user.articles.build(article_params)
+    respond_to do |format|
+      if @article.save
+        if params[:images].present?
```
Comment (charusat09) [inline app/controllers/articles_controller.rb:30]: @uzaif313 can you please extract this code to new private method i.e. save_image(image_param) ?

File: app/controllers/articles_controller.rb [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    end
+  end
+
+  # PATCH/PUT /articles/1
+  # PATCH/PUT /articles/1.json
+  def update
+    respond_to do |format|
+      if @article.update(article_params)
+        if params[:images].present?
+          @article.pictures.destroy_all
```
Comment (charusat09) [inline app/controllers/articles_controller.rb:50]: @uzaif313  same here..

**Comment by charusat09** [comment]
@uzaif313 @hardikupadhyay16 @chaudharyprakash Do we need view template any more ??

**Comment by charusat09** [comment]
@hardikupadhyay16 @uzaif313 @chaudharyprakash also resolve [this issue](https://github.com/charusat09/kms/issues/4) within this branch only..

**Comment by uzaif313** [comment]
this branch is for web not for API

**Comment by charusat09** [comment]
Done

**Pull request closed without merging**